test/realm_test.dart

```diff
@@ -789,33 +791,49 @@ Future<void> main([List<String>? args]) async {
     expect(stored.location.name, 'Europe/Copenhagen');
   });
 
-  test('Realm - open local not encrypted realm with encryption key', () {
-    openEncryptedRealm(null, generateValidKey());
+  test('Realm - encrytpion works', () {
+    var config = Configuration.local([Friend.schema], path: p.join(Configuration.defaultStoragePath, "${generateRandomString(8)}.realm"));
```

Comment:
Why do we need this path? I guess it's because you want different paths for the encrypted/unencrypted realm, but a simpler solution imo would be something like:

```dart
final unencryptedConfig = Configuration.local([Friend.schema]);
final encryptedConfig = Configuration.local([Friend.schema], path: '${unencryptedConfig.path}_encrypted');
```

Reply:
I need a completely new realm that is not the default one. Since I am reading it as a string

Reply:
The default realm is randomly generated for each test, so it should be as good as new.

Reply:
yeah I prefer to be more explicit here. So two new realms with random names created in the test. Names are not relevant much, but the fact, that we are creating them brand new, is relevant.

Reply:
I am not sure what the problem is in the way test is written. As I said I prefer to have this test explicitly set the name of the two files highlighting the fact we are creating a brand new realm and reading it as a Dart string. Also this test was send as an example to the docs team so they can write their own. One more reason I have chosen to set explicitly the path there.